code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1503,6 +1517,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 					H.update_damage_overlays()
 			else//no bodypart, we deal damage with a more general method.
 				H.adjustBruteLoss(damage_amount)
+			if (def_zone == BODY_ZONE_PRECISE_GROIN)
```

Comment:
add a special interaction for the bone codpiece accessory

Reply:
also, was it intentional to make this a 100% chance (if the effective force after armor is high enough)

Reply:
yes